price-estimator/README.md

```diff
@@ -1,18 +1,25 @@
 ## API
 
+All endpoints use the query part of the url with these key-values:
+
+* `atoms`: Required. If set to `true` all amounts are denominated in the smallest available unit (base quantity) of the token. If `false` all amounts are denominated in the "natural" unit of the respective token given by the number of decimals specified through the ERC20 interface. TODO: `false` is currently only implemented for estimated-buy-amount .
```

Comment:
I think it would make sense to default this to false (can be a separate PR given the size of this one).